reconcile/gql_definitions/ocm_subscription_labels/clusters.gql

```diff
@@ -0,0 +1,18 @@
+# qenerate: plugin=pydantic_v1
+
+query OcmLabelClusters {
```

Comment:
Should this be renamed OcmSubscriptionLabelClusters or maybe even remove `Clusters`?

Reply:
Changed in [afafe36](https://github.com/app-sre/qontract-reconcile/pull/3714/commits/afafe3614593e4a131c9c7535445af2c631a7d27)